client/src/styles/index.css

```
@@ -31,3 +31,13 @@ html {
 .custom-scrollbar::-webkit-scrollbar-thumb:hover {
   background: #056896;
 }
+
+:body {
```

Biome (2.1.2)

[error] Unexpected unknown pseudo-class body 

See MDN web docs for more details.

(lint/correctness/noUnknownPseudoClass)